lib/morph/sqlite_diff.rb

```diff
@@ -106,6 +110,12 @@ def self.diffstat_table(table, db1, db2, page = 1000)
         min = [min1, min2].min
         max = [max1, max2].max
       end
+      [min, max]
+    end
+
+    # Page is the maximum number of records that are read into memory at once
+    def self.diffstat_table(table, db1, db2, page = 1000)
```

Comment:
Rather than describing the parameter in a comment I was trying to think of a better name but I can't come up with one :(